docs/instructions.md

```diff
@@ -0,0 +1,29 @@
+# Configuration
+
+ItchySats on the Embassy requires a fully synced Electrum (electrs) node as a source for blockchain data. This requirement will be automatically enforced by EmbassyOS.
+The password can be found in the `Properties` section.
+
+# ItchySats
+
+ItchySats enables peer-2-peer CFD trading on Bitcoin using DLCs (discreet log contracts). No account needed, no trusted third-party - just you and your keys.
+
+This is beta software. We test it on test- and mainnet, but there are no guarantees that it will always behave as expected.
+Please be mindful with how much money you trust the application with.
+
+CFDs trading is inherently risky, be sure to read up on it before using this application.
+
+That said: This is pretty awesome, go nuts!
+
+1. Fund the ItchySats wallet
+2. Open a position
+3. Monitor the price movement
+4. Profit
+
+Limitations of the mainnet beta:
+
+1. Minimum position quantity is $100, maximum $1000
```

Comment:
🔧 This only applies to BTCUSD.

Reply:
We should probably have a central place from where we pull this description, as it was just copy-pasted from Umbrel one.

Reply:
We can create a central description that we can use to copy paste if that is what you mean, but I am not sure it adds such a big advantage (in the end we just have one *more* place to update...).

It would be even better if we could link / embed a central description, but I don't see an easy way to achieve this for all the different setups - Umbrel / embassy wrapper / (...) - and I don't think a link to outside of the respective setup is a good solution.